sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -244,7 +277,7 @@ if (batch.count > 0) {
 }
 ```
 
-### Migrating code from `EventProcessorHost` to `EventHubConsumerClient` for receiving events
+### Migrating from `EventProcessorHost` to `EventHubConsumerClient` for receiving events
```

Comment:
What do you think of moving this under the `Sending events` section as an h4? Same for the "migration samples" for receiving?

Reply:
That seems reasonable, will do.